components/ContextToRuntime.coffee

```diff
@@ -104,21 +112,17 @@ exports.getComponent = ->
           # Different runtime, different graph. Disconnect old runtime connection
           currentContext.runtime.disconnect()
 
+      send = _.debounce sendContext, 300, true
```

Comment:
Buffering until we receive capabilities would seem less racy than a time-based bump...